interface/github/webhook.py

```diff
@@ -0,0 +1,53 @@
+"""Contain all the logic for handling webhooks in a class."""
+import json
+import logging
+
+
+class WebhookHandler:
+    """Encapsulate methods for GitHub webhook triggered events."""
+
+    def __init__(self, db_facade):
+        """Give handler access to the database."""
+        self.__facade = db_facade
+
+    def handle_organization_event(self, payload):

```

Comment:
Mostly because I'm not sure what the best way of hooking up the webhook handler to the facade is, as the core, who is currently already using an instance of the facade comes from a factory. Is it ok to create a new instance of the facade in `server.py`?

Reply:
Oh I see the problem. Hmm, we probably only want one copy of the database facade floating around - it wouldn’t be the worst thing in the world to have multiple, but I don’t think it’s really necessary. Maybe it’s possible to add a method to `core` to retrieve a webhook handler stored in `core`? I’ll have to double check how it’s all structure tomorrow.